polaris-react/src/components/Banner/index.ts

```diff
@@ -1 +1,2 @@
 export * from './Banner';
+export type {BannerHandles} from './utilities';
```

Comment:
This was previously part of the `*` export in Banner, but since I moved it to `utilities`, I have to export it separately